Src/VimCore/CoreInterfaces.fs

```diff
@@ -5157,6 +5174,14 @@ and IVimBuffer =
     [<CLIEvent>]
     abstract Closed: IDelegateEvent<System.EventHandler>
 
+    /// Raised when Call CSharpScript
+    [<CLIEvent>]
+    abstract CallCSharpScript: IDelegateEvent<System.EventHandler<CallCSharpScriptEventArgs>>

```

Comment:
I'm a bit skeptical that we need this. Consider that there is no such event for running a vim script. Feels like we should have an event for both or neither.